docs/GettingStartedDocs/buildandsign.md

```diff
@@ -120,14 +120,17 @@ FamilyID=47183823-2574-4bfd-b411-99ed177d3e43
 ExtendedProductID=2768c720-1e28-11eb-adc1-0242ac120002
 ```
 
-As a convenience, you can specify the enclave properties in code using the `OE_SET_ENCLAVE_SGX2` macro to leverage SGX2 properties and enable KSS using RequireKSS argument in the macro. For example, the equivalent properties could be defined in any .c or .cpp file compiled into the enclave:
+On SGX2 machines, developers can also specify wether to handle in-enclave exceptions or not using `CapturePFGPExceptions` field.
```

Comment:
```suggestion
On SGX2 machines, developers can also specify whether to handle in-enclave exceptions or not using `CapturePFGPExceptions` field.
```

Reply:
@manojrupireddy , would it be possible to update oesign's README as @RRathna has stated above. After that, I can merge this.